engine/buildkit/executor_spec.go

```diff
@@ -346,7 +346,7 @@ func (w *Worker) generateBaseSpec(ctx context.Context, state *execState) error {
 		w.idmap,
 		w.apparmorProfile,
 		w.selinux,
-		w.tracingSocket,
+		"",
```

Comment:
By passing in `""` we don't need to deal with Buildkit's configured `OTEL_` env vars, and can just set our own.